pages/product/index.vue

```diff
@@ -361,6 +370,10 @@ export default defineComponent({
       return productGetters.getVariationProductCodeOrProductCode(product.value)
     })
 
+    const isLoadingWishlist = computed(() => {
+      return loadingWishlist.value || cartLoading.value
```

Comment:
cartLoading should not be inside PDP

Reply:
You are right, good catch, It has it's use in cart. Resolved.